tools/ci_build/build.py

```diff
@@ -938,7 +965,8 @@ def run_training_python_frontend_e2e_tests(args, cwd, dll_path):
         cwd=cwd, dll_path=dll_path)
 
 def run_onnxruntime_tests(args, source_dir, ctest_path, build_dir, configs,
-                          enable_tvm=False, enable_tensorrt=False):
+                          enable_python_tests, enable_tvm=False,
```

Comment:
enable_python_tests was removed from ort_training here: https://aiinfra.visualstudio.com/Lotus/_git/onnxruntime/pullrequest/5575?_a=overview

i think we probably don't want to reintroduce it here